.travis.yml

```diff
@@ -23,10 +24,11 @@ matrix:
       env: DEPENDENCIES='dev' SYMFONY_VERSION='3.4.*@dev'
     - php: 7.1
       env: DEPENDENCIES='dev' SYMFONY_VERSION='4.0.*'
+    - php: 7.2
+      env: DEPENDENCIES='dev' SYMFONY_VERSION='4.0.*'
   fast_finish: true
 
 before_script:
-  - if [ "$SYMFONY_VERSION" != "" ]; then composer require --no-update "symfony/framework-bundle:${SYMFONY_VERSION}"; fi;
```

Comment:
This line should be kept as there're different SF versions to test in the matrix.